File: AsyncTask/app/src/main/java/com/example/rodolfo/asynctask/MainActivity.java
package com.example.rodolfo.asynctask;

import android.graphics.Bitmap;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

public class MainActivity extends AppCompatActivity implements TarefaInterface {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void downloadImg(View view){
        Tarefa tarefa =  new Tarefa(this,this);
        tarefa.execute("http://www.thiengo.com.br/img/system/logo/thiengo-80-80.png");
    }

    @Override
    public void inserirMainThread(Bitmap img) {
        ImageView iv =  (ImageView) findViewById(R.id.imageView);
        iv.setImageBitmap(img);
    }
}

File: DeltaPratoDoDia/app/src/main/java/com/example/rodolfo/deltapratododia/MainActivity.kt
package com.example.rodolfo.deltapratododia

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.support.v7.widget.LinearLayoutManager
import android.support.v7.widget.RecyclerView
import android.support.v7.widget.Toolbar

class MainActivity : AppCompatActivity() {
    internal var mToolbar: Toolbar
    internal var mRecycler: RecyclerView
    internal var mAdapter: RecyclerView.Adapter<*>
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        mToolbar = findViewById(R.id.my_toolbar) as Toolbar
        setSupportActionBar(mToolbar)

        mRecycler = findViewById(R.id.my_recycler_view) as RecyclerView
        mRecycler.setHasFixedSize(true)
        mRecycler.layoutManager = LinearLayoutManager(LinearLayoutManager.VERTICAL)
        mAdapter = MyAdapter()
        mRecycler.adapter = mAdapter
    }
}